applications/nrf_desktop/doc/leds.rst

```diff
@@ -0,0 +1,43 @@
+.. leds:
+
+leds module
+###########
+
+``leds`` module is responsible for controlling LED diodes (setting LED effect in reaction to ``led_event``).
+
+
+Configuration
+*************
+
+``leds`` module is based on Zephyr PWM driver.
+In order to use the module, following configuration options have to be set:
+
+- :option:`CONFIG_PWM`
+- :option:`CONFIG_PWM_0`
+  0 denotes PWM port number. All PWM ports used by application should be configured.
+
+For current implementation :option:`CONFIG_PWM_NRFX` is set and :option:`CONFIG_PWM_NRF5_SW` is unset (in order to use hardware PWM driver).
+
+Module is enabled and configured using Kconfig options (details described as help).
+
+File ``leds_def.h`` is used to define mapping PWM channels to pin numbers.
+Current implementation uses symbols defined in device tree.
+These symbols are defined for board, but application may overlay definining own ``dts.overlay`` file).
```

Comment:
``dts.overlay`` file in the application configuration directory dedicated for this board.

also remove `)` - it does not match anything

Reply:
done